assets/app/views/_overview-deployment.html

```diff
@@ -0,0 +1,84 @@
+<!--
+  Expects the following variables. All must be defined in the overviewDeployment directive scope.
+
+  rc   - ReplicationController. Required. May or may not have deployment annotations.
+  pods - [Pod]. Required.
+
+  deploymentConfigId               - String. Optional. If present, indicates this replication controller came from a deployment.
+  deploymentConfig                 - DeploymentConfig. Optional.
+  deploymentConfigDifferentService - Boolean. Optional.
+
+  imagesByDockerReference - map[dockerReference][]Image. Optional.
+  builds                  - map[buildId]build. Optional.
+ -->
+<div class="osc-object components-panel deployment-block" 
+     kind="ReplicationController" resource="rc"
+     ng-init="isDeployment = (deploymentConfigId && (rc | annotation:'deploymentConfig') && (rc | annotation:'deploymentVersion'))">
+
+  <!--
+  Connector to the containing service block.
+  Hidden by CSS when in non-service blocks.
+  -->
+  <div class="connector">
+    <i class="fa fa-search"></i>
```

Comment:
yes, the grouping connector is made of the search icon (rotated) and overlaying a circle icon, both positioned vertically in center of the deployment block. :)
![connector-point](https://cloud.githubusercontent.com/assets/1874151/7727089/9ba8a2c8-fed4-11e4-91f2-2ee69bccd10c.png)